<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CCtrBar</class>
 <widget class="QWidget" name="CCtrBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>45</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QSlider" name="Slider_range">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>196</width>
     <height>42</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_record">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="CtrBar.qrc">
        <normaloff>:/icon/icon/record.png</normaloff>:/icon/icon/record.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_play">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="CtrBar.qrc">
        <normaloff>:/icon/icon/play.png</normaloff>:/icon/icon/play.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_pause">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="CtrBar.qrc">
        <normaloff>:/icon/icon/pause.png</normaloff>:/icon/icon/pause.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_stop">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="CtrBar.qrc">
        <normaloff>:/icon/icon/stop.png</normaloff>:/icon/icon/stop.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="CtrBar.qrc"/>
 </resources>
 <connections/>
</ui>
